glib/src/variant.rs

```diff
@@ -642,7 +646,22 @@ where
     V: FromVariant,
     H: BuildHasher + Default,
 {
+    /// ```
```

Comment:
Maybe turn it into a code comment? Inner functions are not documented after all. :)

Reply:
Will it still be picked up as a test?

Reply:
No, but you could also add a non-doctest here

```rust
#[test]
fn test_that_thing() {
    [...]
}
```